sanic/base.py

```diff
@@ -16,9 +16,7 @@ def __init__(self, *args, **kwargs):
             nonlocal init
             nonlocal name
 
-            bases = [
-                b for base in type(self).__bases__ for b in base.__bases__
-            ]
+            bases = BaseSanic.__bases__
 
             for base in bases:
```

Comment:
Since we no longer need the comprehension, no need to have `bases` as a variable.

```python
for base in BaseSanic.__bases__:
```